### DISCUSSION TITLE
Add Customer

### DISCUSSION THREAD

**Discussion by YunusEmreAlps** [uvdesk/community-skeleton#595] [Q&A]
Hello,

I hope you're well and everything is good for you.

I want to add a couple of inputs (company info, etc.) for the user profile. 

Updated these files:
createSupportCustomer.html.twig
Entity/User.php
Form/UserForm.php

Can I add a column to the DB table or when I run the code auto-added? 
Which files must be updated for this process?

Thank you very much.
Best regards

![Screenshot (559)](https://user-images.githubusercontent.com/44679555/186398299-42439678-2a29-4035-be65-fa27b3206ada.png)
,

**Comment by PeopleInside** [comment]
Hi, adding custom field is not included in the open source version but can be done with an additional component:
- [UVdesk Open Source Custom Fields App](https://store.webkul.com/uvdesk-open-source-custom-fields-app.html) ($99.00)

More info about additional app can be found here: https://forums.uvdesk.com/topic/2010/additional-apps-that-you-can-add-to-the-open-source-uvdesk-helpdesk-system

I hope this helps.

**Reply by YunusEmreAlps** [reply]
I want to manage this customer list. For example, I want to show the full name of the customer.

How can I manage this? Which file?

![Screenshot (563)](https://user-images.githubusercontent.com/44679555/187668830-342157f4-924c-4c37-9a46-b949c36e17e2.png)

Update Note: This problem is solved 

Thank you very much.
Best regards

**Reply by PeopleInside** [reply]
I don't think there is actually an option for this but we need a confirmation from the UVdesk staff.

**Reply by YunusEmreAlps** [reply]
Default Ticket Label View Template Not translated very well.

![Screenshot (567)](https://user-images.githubusercontent.com/44679555/188562446-69eb3a56-ec0a-4342-b744-82d2c0585821.png)

![Screenshot (569)](https://user-images.githubusercontent.com/44679555/188563051-7cba5cc4-0517-4763-a7ea-a7c77972d877.png)


Thank you very much.
Best regards

**Comment by PeopleInside** [comment]
@YunusEmreAlps if you find bad translation or bugs you need open an issue here:
https://github.com/uvdesk/community-skeleton/issues
